eos-tech-support/eos-convert-system-qa

```diff
@@ -93,14 +93,14 @@ if $OSTREE || $APPS; then
 
     # Change the Flatpak runtime and apps server URLs.
     if $APPS; then
-        runtime_url=$(flatpak remote-list -d | awk '/^eos-runtimes/{print $3}')
+        runtime_url=$(ostree config get 'remote "eos-runtimes".url')
```

Comment:
Great. There is one little issue, though. If this is a split disk, a different repo is used. Can you pass the repo explicitly? Something like this:
```
if [ -d /var/endless-extra/flatpak/repo ]; then
    flatpak_repo=/var/endless-extra/flatpak/repo
else
    flatpak_repo=/var/lib/flatpak/repo
fi
runtime_url=$(ostree --repo="$flatpak_repo" config get 'remote "eos-runtimes".url')
...
```
And continue using `--repo="$flatpak_repo"` while operating on the apps.